cpp/src/parquet/properties.h

```diff
@@ -351,36 +408,64 @@ class PARQUET_EXPORT WriterProperties {
       return this->compression_level(path->ToDotString(), compression_level);
     }
 
+    /**
+     * Define the file encryption properties.
+     * Default NULL.
+     */
     Builder* encryption(
         std::shared_ptr<FileEncryptionProperties> file_encryption_properties) {
       file_encryption_properties_ = std::move(file_encryption_properties);
       return this;
     }
 
+    /**
+     * Enable statistics in general.
+     * Default enabled.
+     */
     Builder* enable_statistics() {
       default_column_properties_.set_statistics_enabled(true);
       return this;
     }
 
+    /**
+     * Disable statistics in general.
+     * Default enabled.
+     */
     Builder* disable_statistics() {
       default_column_properties_.set_statistics_enabled(false);
       return this;
     }
 
+    /**
+     * Enable statistics for the column specified by `path`.
+     * Default enabled.
+     */
     Builder* enable_statistics(const std::string& path) {
       statistics_enabled_[path] = true;
       return this;
     }
 
+    /**
+     * Enable statistics for the column specified by `path`.
+     * Default enabled.
+     */
     Builder* enable_statistics(const std::shared_ptr<schema::ColumnPath>& path) {
       return this->enable_statistics(path->ToDotString());
     }
 
+    /**
+     * Disable statistics for the column specified by `path`.
+     * Default enabled.
+     */
     Builder* disable_statistics(const std::string& path) {
       statistics_enabled_[path] = false;
       return this;
     }
 
+    /**
+     * Disable statistics for the column specified by `path`.
+     * Default enabled.
+     */
     Builder* disable_statistics(const std::shared_ptr<schema::ColumnPath>& path) {
       return this->disable_statistics(path->ToDotString());
     }
```

Comment:
Likewise, it seems `build()` below could use a doc comment (just to make it show up in Sphinx)?

Reply:
Yes. The reason for this PR is to make them show up in Sphinx so further also in doc. Then future readers of the doc wont miss those tuning parameters :).